### PR TITLE
Feat/add primary btn

### PR DESCRIPTION
**Pull request by imadatyatalah**
~Mazal khass icons~
Icons are done

### CODE REVIEW

**Review by Stormix — APPROVED** [review]
LGTM! Follow up with @smakosh for some recommendations about css in js.

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import { styled } from "../stitches.config";
+
+const Icon = styled("span", {
+  display: "inline-flex",
+});
+
+const ButtonStyle = styled("button", {
```
Comment (smakosh) [inline src/components/Button.js:9]: `styled.button`

Comment (imadatyatalah) [reply src/components/Button.js:9]: `styled.button` kant f alpha version db stitches ra f beta o 7ydo support l proxy
take a look at ["No more proxy" section ](https://stitches.dev/blog/migrating-from-alpha-to-beta)

Comment (smakosh) [reply src/components/Button.js:9]: I'm talking about styled-component mashi stitches, styled-system is way better https://styled-system.com/

Comment (imadatyatalah) [reply src/components/Button.js:9]: I think dkchi li andiroh b styled-system ymkn ndiroh b [stitches utils](https://stitches.dev/docs/utils)

Comment (smakosh) [reply src/components/Button.js:9]: Yes gha styled-system m3a SC complement each other better

**Pull request merged**